Description: fix compile when openssl doesn't support ssl2
Origin: upstream, http://websvn.kde.org/?view=revision&revision=1230301

--- qca-ossl/qca-ossl.cpp
+++ qca-ossl/qca-ossl.cpp
@@ -5235,9 +5235,11 @@
 		OpenSSL_add_ssl_algorithms();
 		SSL_CTX *ctx = 0;
 		switch (version) {
+#ifndef OPENSSL_NO_SSL2
 		case TLS::SSL_v2:
 			ctx = SSL_CTX_new(SSLv2_client_method());
 			break;
+#endif
 		case TLS::SSL_v3:
 			ctx = SSL_CTX_new(SSLv3_client_method());
 			break;
